src/nvim/lua/vim.lua

```diff
@@ -779,4 +407,6 @@ function vim._log_keystroke(char)
   end
 end
 
+vim.opt = require('vim.opt')
```

Comment:
having the same module manipulate `vim.` directly and return one (1) key to add to `vim.` is a bit suspicious. probably better to load it like `vim/global.lua` i e only mutate `vim.` in place.

Reply:
yes, definitely. I think I will return other stuff from it when I'm done (also, not everything there is options anymore. I may change to something like `meta` because they are all special metatables about interacting with vim's state).